recipe/meta.yaml

```diff
@@ -13,22 +13,22 @@ source:
     - no_crt_copy.patch    # [win]
     - no_snprintf.patch    # [win and py35]
     - vs2010_openmp.patch  # [win and py34]
+    - vs2010_path.patch    # [win64 and py34]
 
 build:
-  number: 0
+  number: 1
   features:
     - vc9     # [win and py27]
     - vc10    # [win and py34]
     - vc14    # [win and py35]
-  skip: True  # [win]
 
 requirements:
   build:
     - python  # [win]
 
 test:
   requires:
-    - python {{ environ['PY_VER'] + '*' }}                # [win]
+    - python                                              # [win]
```

Comment:
> I did remove the conda-build pinning to get it to build in the first place.

Ah, I see. I guess it is hard to tell as `appveyor.yml` was added in this PR too.

> I though the same at first but it appears to work now...

Still makes me a little nervous, but it is not a blocker from me. This only affects tests not the package itself.